pkg/manifests/enable_patch_policy.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
this and the other yamls under pkg/manifests seem duplicate to the ones in the examples/poc folder, do we need both?

Reply:
Yeah, you're absolutely right. I think eventually we just want to delete the poc directory. But while we are still using forks of vLLM to operate. I think keeping the poc folder and a way to set it up there kind of implies the WIP nature of our project.

Or we can always fast-follow this PR with removing the poc dir

LMKWYT!